### PR TITLE
Log cacheability

### PR DESCRIPTION
**Pull request by gquintard**
Ham-fisted attempt at logging cacheability

### CODE REVIEW

**Comment by bsdphk** [comment]
Apart from wondering if "pass" is the right thing to log for "uncacheable" I'm OK with this.

**Comment by lkarsten** [comment]
Is it wise to add another meaning of "pass" in Varnish?

cacheable | uncacheable? too long?

**Comment by gquintard** [comment]
(un)cacheable seemed a bit long, but I have no problem with changing it.

**Comment by nigoroll** [comment]
I had a hard time with the `uncacheable` attribute in VCL, I would have preferred if we only used `pass` instead. But I understand that `uncacheable` combines all the different cases (`pass`, `hitpass`, `hitmiss`, make uncacheable) so I think that, as we once decided to go with `uncacheable`, the right output choce would be `uncacheable` and `cacheable`

File: bin/varnishd/cache/cache_rfc2616.c [hunk 1]
```diff
@@ -198,9 +198,10 @@ RFC2616_Ttl(struct busyobj *bo, double now, double *t_origin,
 	}
 
 	VSLb(bo->vsl, SLT_TTL,
-	    "RFC %.0f %.0f %.0f %.0f %.0f %.0f %.0f %u",
+	    "RFC %.0f %.0f %.0f %.0f %.0f %.0f %.0f %u %s",
 	    *ttl, *grace, *keep, now,
-	    *t_origin, h_date, h_expires, max_age);
+	    *t_origin, h_date, h_expires, max_age,
+	    bo->uncacheable ? "pass" : "cache");
```
Comment (nigoroll) [inline bin/varnishd/cache/cache_rfc2616.c]: `bo->uncacheable ? "uncacheable" : "cacheable");`

**Comment by gquintard** [comment]
Will commit with cacheable/uncacheable on Thursday if no one yells before then

**Comment by gquintard** [comment]
closed via 9e174d4c1d38b752e029d25c28127f9eeace39c9

**Pull request closed without merging**